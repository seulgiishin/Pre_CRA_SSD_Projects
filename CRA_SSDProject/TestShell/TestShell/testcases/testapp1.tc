{
	"ID": "TestApp1",
	"STEPS":[
		"fullwrite 0xAABBCCDD",
		"fullread"
	],	
	"EXPECTED": [
		"testapp1.expected"
	]
}