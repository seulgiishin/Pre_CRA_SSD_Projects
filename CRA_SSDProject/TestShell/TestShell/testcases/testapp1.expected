0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD
0xAABBCCDD